sphinx/screens-detail/subscription/subscription/src/main/java/chat/sphinx/subscription/ui/SubscriptionViewState.kt

```diff
@@ -0,0 +1,16 @@
+package chat.sphinx.subscription.ui
+
+import chat.sphinx.wrapper_subscription.Subscription
+import io.matthewnelson.concept_views.viewstate.ViewState
+
+internal sealed class SubscriptionViewState: ViewState<SubscriptionViewState>() {
+    object Idle: SubscriptionViewState()
+
+    object SavingSubscription: SubscriptionViewState()
+
+    object SavingSubscriptionFailed: SubscriptionViewState()
+
+    class SubscriptionLoaded(
+        val subscription: Subscription
```

Comment:
subscription data does not describe the view's state. move all logic to view model.